h5py/tests/test_slicing.py

```diff
@@ -316,3 +316,103 @@ def test_write_noncompound(self):
         data2['b'] = 1.0
         self.dset['b'] = 1.0
         self.assertTrue(np.all(self.dset[...] == data2))
+
+
+class TestMultiBlockSlice(BaseSlicing):
+
+    def setUp(self):
+        super(TestMultiBlockSlice, self).setUp()
+        self.arr = np.arange(10)
+        self.dset = self.f.create_dataset('x', data=self.arr)
+
+    def test_default(self):
+        # Default selects entire dataset as one block
+        mbslice = MultiBlockSlice()
+
+        self.assertEqual((0, 10, 1, 1), mbslice.indices(10))
+        np.testing.assert_array_equal(self.arr, self.dset[mbslice])
```

Comment:
```suggestion
        self.assertEqual(mbslice.indices(10), (0, 10, 1, 1))
        np.testing.assert_array_equal(self.dset[mbslice], self.arr)
```

It's cosmetic, but I've glanced at a few of our existing tests, and it looks like we generally stick to `(result, expected)` rather than `(expected, result)`, so I'd like to keep it consistent.

(And the same through the additions in this file)